### PR TITLE
Gardening: Migrate test suite to GH issues p. 26

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

File: validation-test/compiler_crashers_2_fixed/0069-sr3657.swift [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend %s -emit-ir
 
+// https://github.com/apple/swift/issues/46442
+
```
Comment (AnthonyLatsis) [inline validation-test/compiler_crashers_2_fixed/0069-sr3657.swift:4]: Typo in file name: the correct SR is 3<ins>8</ins>57 (see https://github.com/apple/swift/pull/7304)

**Comment by AnthonyLatsis** [comment]
Note: The files that are not recognized as renamed when modified simultaneously are first modified and then renamed in a separate commit.

File: validation-test/compiler_crashers_2_fixed/0069-issue-46442.swift [hunk 2]
```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend %s -emit-ir
 
+// https://github.com/apple/swift/issues/46442
+
```
Comment (AnthonyLatsis) [inline validation-test/compiler_crashers_2_fixed/0069-issue-46442.swift:4]: Typo in file name: the correct SR is 3<ins>8</ins>57 (see https://github.com/apple/swift/pull/7304)

**Review by xedin — APPROVED** [review]

**Comment by xedin** [comment]
@swift-ci please test

**Comment by AnthonyLatsis** [comment]
@swift-ci please test

**Pull request merged**